wwwroot/cgi-bin/lib/robots.pm

```diff
@@ -400,26 +404,26 @@
 'bingpreview',
 'MSIECrawler',
 'msnbot/',
-'msnbot\-media/',
+'msnbot-media/',
 'AdIdxBot/',
 'NOT[\x20]Googlebot/',
 'Googlebot/',
 'Google[\x20]Web[\x20]Preview',
-'Googlebot\-Image/',
-'Googlebot\-Mobile/',
-'google\-sitemaps',
-'Googlebot\-News',
-'Googlebot\-Video/',
-'AdsBot\-Google[\x20]\(',
-'AdsBot\-Google\-Mobile\-Apps',
+'Googlebot-Image/',
+'Googlebot-Mobile/',
+'google-sitemaps',
+'Googlebot-News',
+'Googlebot-Video/',
+'AdsBot-Google[\x20]\(',
+'AdsBot-Google-Mobile-Apps',
 'Mediapartners-Google',
-'Feedfetcher\-Google',
-'Google\-Adwords\-Instant',
+'Feedfetcher-Google',
+'Google-Adwords-Instant',
 'Firefox/1\.5',
 'Yahoo![\x20]Slurp[\x20]China',
 'Yahoo![\x20]Slurp',
 'Baiduspider/',
-'Baiduspider\-image',
+'Baiduspider-image',

```

Comment:
Regarding the Device Pixel Ratio change, There are Apple monitors which have screen.width = 2560px and have a DPR of 1. And there are other Apple monitors which have screen.width = 2560 and have DPR of 2. They are very different monitors, the one with DPR =2 being a 5K monitor with twice the resolution. However the pixel doubling (DPR = 2 ) that the 5K monitor uses casues it to return a screen.width of 5120/DPR = 2560px to the DOM screen.width so currently we don't know if its a 5120 (5K) monitor or a 2560 monitor since they both use screen.width of 2560..
So now that there are many devices, especially mobile phones using a dpr of between 1 and upto 6 (so far I think) the screen sizes report doesn't give a true picture of what devices are being used.
I think people would be more interested in seeing what the dpr is which I realise is a significant change to the log output file and would require including screen dimension combined with dpr as the unique identifier which would make the screen size report longer as there would be more unique combinations than there are currently. The current screen size report is not accurate becasue of this since its not telling us which true monitor resolutions are being used. 
As an interim I just added the dpr into the query string so anyone who wants to check (me) can see it in the raw log files. It has no effect on reporting as it currently stands.